### PR TITLE
Claim addresses.

### PR DESCRIPTION
**Pull request by bishopbm1**
Added test_ip_address and test_range actions and workflows. Updated dns_record_claim to test ranges and ip_addresses before assignment. Found issue with soap call sending incorrect param. Added an override for this case and a test case as well. bumped pack version and changelog.

### CODE REVIEW

File: actions/workflows/dns_record_claim.yaml [hunk 1]
```diff
@@ -97,6 +109,20 @@ tasks:
...
+  # Verify the IP address is available for use
+  test_ip_address:
+    action: menandmice.test_ip_address
+    input:
+      server: "{{ ctx().server }}"
+      session: "{{ ctx().session }}"
+      transport: "{{ ctx().transport }}"
+      range_name: "{{ ctx().range_name }}"
+      ip_address: "{{ ctx().ip_address }}"
+    next:
```
Comment (nmaludy) [inline actions/workflows/dns_record_claim.yaml:121]: what happens if this fails?

File: pack.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ keywords:
   - mice
   - ipam
   - dns
-version: 1.0.2
+version: 1.0.3
```
Comment (nmaludy) [inline pack.yaml]: deserves a bigger version bump

Comment (bishopbm1) [reply pack.yaml]: the workflow fails. Nothing has been claimed yet so no remediation is needed

File: actions/workflows/dns_record_claim.yaml (hunk 1 above)
Comment (bishopbm1) [reply actions/workflows/dns_record_claim.yaml:121]: the workflow fails. Nothing has been claimed yet so no remediation is needed

**Pull request merged**